util/pkg/vfs/s3fs.go

```diff
@@ -61,6 +61,14 @@ func newS3Path(s3Context *S3Context, bucket string, key string) *S3Path {
 }
 
 func (p *S3Path) Path() string {
+	response,_ := p.GetVersioning()
```

Comment:
I am thinking we want the warning here:

https://github.com/homernajafi/kops/blob/5df03c5ac09d9595ca77bd820d7398c75f604f2d/util/pkg/vfs/context.go#L223